tests/python/unittest/test_operator.py

```diff
@@ -8054,6 +8054,68 @@ def test_image_normalize():
     # check backward using finite difference
     check_numeric_gradient(img_norm_sym, [data_in_4d], atol=0.001)
 
+@with_seed()
+def test_index_array_default():
+    for shape in [(10,), (7, 5, 29), (5, 7, 11, 13, 17, 19)]:
+        data  = mx.symbol.Variable("data")
+        index_array = mx.sym.contrib.index_array(data)
+
+        input_array = np.ones(shape)
+        mgrid = np.mgrid[tuple(slice(0, x) for x in shape)]
+        expected = np.stack(mgrid, axis=-1)
+
+        check_symbolic_forward(index_array, [input_array], [expected])
+        check_symbolic_backward(index_array, [input_array], [np.ones(expected.shape)], [np.zeros_like(input_array)])
+
+@with_seed()
+def test_index_array_default_zero_dim():
+    with mx.np_compat(active=True):
```

Comment:
If the whole function is expected to be numpy compatible, you can use a decorator `mx.use_np_compat` to wrap the whole function. It would be easier for us to just delete the decorator in the next major release than putting additional efforts of adjusting indentation.

Reply:
Thanks, I've updated the subtests to use `@mx.use_np_compat` in 139567b9e5d1cf909b69355e0aeb162e38eb5ed6.